tests/cases/conformance/parser/ecmascriptnext/numericSeparators/parser.numericSeparators.hex.ts

```diff
@@ -0,0 +1,4 @@
+0x00_11;
+0X0_1;
+0x1100_0011;
+0X0_11_0101;

```

Comment:
Per https://github.com/tc39/proposal-numeric-separator/issues/25 it looks like "\u{10_ffff}" is permitted as an extended Unicode escape sequence. Can you add this to your existing tests?

Reply:
Discussion on https://github.com/tc39/proposal-numeric-separator/issues/25 has continued and it is apparently now an explicit syntax error to witness an `_` in a literal in an escape sequence.